src/lib/services/index.ts

```diff
@@ -490,6 +490,7 @@ export const createServices = (
         projectStatusService,
         transactionalUserSubscriptionsService,
         uniqueConnectionService,
+        featureLifecycleReadModel,
```

Comment:
we don't have separate grouping of read models and services so adding it to services. controller uses the read model directly since it's read only path

Reply:
Hi, just FYI, I'm modifying this, as this read model is a store, not a service. So with ESM it messes things a bit up, but I'll just explicitly add the read model dependency, not through the services interface.

Reply:
I think there was a problem with testing since tests access either stores and services and we don't have another grouping for read models

Reply:
The biggest problem was that when we're creating controllers, we only have access to services interface and the db. If we start using read models for simple cases I think it makes sense to also have access to stores (as read models are currently exposed there), then it's a matter of propagating stores to the controller constructor. Does it make sense? 

Alternatively, we could have a list of read models (neither stores nor services)

Reply:
Yeah, we can piggyback on the stores we have too.